Sources/StreamChat/Database/DTOs/ChannelDTO.swift

```diff
@@ -446,6 +446,11 @@ extension ChatChannel {
 
             let allUnreadMessages = currentUserRead?.unreadMessagesCount ?? 0
 
+            // Therefore, no unread messages with mentions and we can skip the fetch
+            if allUnreadMessages == 0 {
+                return .noUnread
+            }
```

Comment:
Below we used to do an additional fetch request for unread messages mentioning the current user. That is not needed if there are no unread messages anyway.